docs/source/usage/plugins/radiation.rst

```diff
@@ -142,38 +142,30 @@ By using ``NyquistFactor = 0.5`` for periodic boundary conditions, particles tha
 Form factor
 """""""""""
 
-The *form factor* is still an experimental method trying to consider the shape of the macro particles when computing the radiation.
-By default, it should be switched off by setting ``__COHERENTINCOHERENTWEIGHTING__`` to zero. 
-
-.. code:: cpp
-
-   // correct treatment of coherent and incoherent radiation from macroparticles
-   // 1 = on (slower and more memory, but correct quantitative treatment)
-   // 0 = off (faster but macroparticles are treated as highly charged, point-like particle)
-   #define __COHERENTINCOHERENTWEIGHTING__ 0
-
+The *form factor* is a method, which considers the shape of the macro particles when computing the radiation.
 
 If switched on, one can select between different macro particle shapes. 
-Currently three shapes are implemented.
+Currently eight shapes are implemented.
 A shape can be selected by choosing one of the available namespaces:
 
 .. code:: cpp
 
    /* choosing the 3D CIC-like macro particle shape */
-   namespace radFormFactor_selected = radFormFactor_CIC_3D;
-
-
-============================ ===================================================================================================================
-Namespace                    Description
-============================ ===================================================================================================================
-``radFormFactor_CIC_3D``     3D Cloud-In-Cell shape
-``radFormFactor_CIC_1Dy``    Cloud-In-Cell shape in y-direction, dot like in the other directions
-``radFormFactor_incoherent`` forces a completely incoherent emission by scaling the macro particle charge with the square root of the weighting
-============================ ===================================================================================================================
+   namespace radFormFactor = radFormFactor_CIC_3D;
 
-.. note:
 
-   possibly more shapes (f.e. spaghetti shape) will be added
+==================================== ===================================================================================================================
+Namespace                            Description
+==================================== ===================================================================================================================
+``radFormFactor_CIC_3D``             3D Cloud-In-Cell shape
+``radFormFactor_TSC_3D``             3D Triangular shaped density cloud
+``radFormFactor_PCS_3D``             3D Quadratic spline density shape
```

Comment:
Optional:
Quadratic as according to Birdsall correct but Parabolic would be also correct and fitting to the abbreviation.

Reply:
optional:
```suggestion
``radFormFactor_PCS_3D``             3D Parabolic spline density shape
```

Reply:
Doesn't PCS stand for Piecewise Cubic Spline?

Reply:
In Hockney (which Birdsall references) this is called PQS (probably for piecewise quadratic spline).

Reply:
@ax3l and @psychocoderHPC why do we use the abbreviation PCS?

Reply:
@finnolec I think your version with quadratic is clearer - keep it.

Reply:
I just took the full name from https://doi.org/10.1016/j.nima.2018.02.020, but didn't change PCS to QSC :D Maybe I should change it to Piecewise quadratic spline

Reply:
That would be best :+1:

Reply:
Btw our README.md says PSQ, but in the code it is PCS. So it's all a mess.